application/src/main/java/org/togetherjava/tjbot/commands/moderation/AuditCommand.java

```diff
@@ -139,26 +138,58 @@ private boolean handleChecks(@NotNull Member bot, @NotNull Member author,
         return ModerationUtils.handleHasAuthorRole(ACTION_VERB, hasRequiredRole, author, event);
     }
 
-    private void auditUser(@NotNull User user, @NotNull ISnowflake guild,
-            @NotNull IReplyCallback event) {
-        List<ActionRecord> actions =
-                actionsStore.getActionsByTargetAscending(guild.getIdLong(), user.getIdLong());
+    private Message auditMessage(long guildId, long targetId, int pageNo, @NotNull JDA jda) {
```

Comment:
the method name reads odd. you cant "audit a message". you can "audit an user" though. I would keep this as `auditUser` or rename it to `createAuditMessageForUser` or similar.

Reply:
fixed